Resources/Prototypes/_NF/Catalog/Fills/Backpacks/duffelbag.yml

```diff
@@ -9,3 +9,25 @@
       - id: WeaponLauncherChinaLakeEmp
       - id: GrenadeEmp
         amount: 8
+
+- type: entity
+  id: ClothingBackpackDuffelScienceBundle
+  parent: ClothingBackpackDuffelScience
+  suffix: Science Kit
+  components:
+    - type: StorageFill
+      contents:
+        - id: SheetGlass1
+          amount: 14
+        - id: SheetPGlass1
+          amount: 10
+        - id: SheetPlasma1
+          amount: 5
+        - id: Beaker
+          amount: 4
+        - id: MatterBinStockPart
+          amount: 10
+        - id: CapacitorStockPart
+          amount: 8
+        - id: MicroManipulatorStockPart
+          amount: 5
```

Comment:
does this all fit in one duffel bag?

Reply:
Yes, had to remove the 2 stacks of steel and wires to the crate to make sure.